cpp/generate.rb

```diff
@@ -28,6 +28,16 @@
             match: /,/,
             tag_as: "comma punctuation.separator.delimiter"
         )
+    # TODO maybe change this tag see https://github.com/jeff-hykin/cpp-textmate-grammar/pull/135#issuecomment-490727262
+    # TODO eventually move this outside of the # Utils section
+    ref_deref_definition_pattern = newPattern(
+            match: zeroOrMoreOf(/\*/.maybe(@spaces)),
+            tag_as: "keyword.operator.dereference"
```

Comment:
that's a pointer, not a dereference.

Reply:
True. Okay, I guess if were going to make it accurate we should decide on something other than `keyword.operator`. Do you think `storage.modifier.pointer` and `storage.modifier.reference` are accurate?